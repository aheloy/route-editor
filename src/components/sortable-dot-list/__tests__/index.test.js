import React from 'react';

import SortableDotList from '../';


const dots = [
  {
    text: '1'
  },
  {
    text: '2'
  },
  {
    text: '3'
  }
];

const onAdd = () => 0;
const onDelete = () => 0;
const onSort = () => 0;


test('render a SortableDotList', () => {
  const wrapper = shallow(
    <SortableDotList
      dots={dots}
      onAdd={onAdd}
      onDelete={onDelete}
      onSort={onSort}
    />
  );
  expect(wrapper).toMatchSnapshot();
});
