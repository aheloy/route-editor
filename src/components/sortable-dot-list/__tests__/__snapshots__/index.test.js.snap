// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a SortableDotList 1`] = `
<div>
  <styled.input
    onChange={[Function]}
    onKeyPress={[Function]}
    placeholder="Новая точка маршрута"
    type="text"
    value=""
  />
  <SortableList
    items={
      Array [
        Object {
          "text": "1",
        },
        Object {
          "text": "2",
        },
        Object {
          "text": "3",
        },
      ]
    }
    onDelete={[Function]}
    onSort={[Function]}
  />
</div>
`;
